examples.md

```diff
@@ -80,12 +84,12 @@ Using [NuGet lock files](https://docs.microsoft.com/nuget/consume-packages/packa
 ## Swift, Objective-C - Carthage
 
 ```yaml
-uses: actions/cache@preview
-      with:
-        path: Carthage
-        key: ${{ runner.os }}-carthage-${{ hashFiles('**/Cartfile.resolved') }}
-        restore-keys: |
-          ${{ runner.os }}-carthage-
+- uses: actions/cache@preview
```

Comment:
This was fixed in https://github.com/actions/cache/pull/57, not needed for this PR